igvm_defs/src/lib.rs

```diff
@@ -662,9 +662,25 @@ pub struct IgvmPageDataFlags {
     pub is_2mb_page: bool,
     /// This page data should be imported as unmeasured.
     pub unmeasured: bool,
+    /// This page data should be imported as assigned but host visible. The page
+    /// contents must be preserved, but are not part of the launch measurement.
+    ///
+    /// NOTE: This is technically unstable, but macro errors prevent us from
+    /// hiding this definition.
+    pub shared: bool,
     /// Reserved.
-    #[bits(30)]
+    #[bits(29)]
     pub reserved: u32,
+    // TODO: Macro errors prevent us from using the desired definition below.
+    // bitfield_struct issue?
+    // #[cfg(feature = "unstable")]
+    // #[cfg_attr(docsrs, doc(cfg(feature = "unstable")))]
+    // pub shared: bool,
```

Comment:
Why don't we just remove these members?

Reply:
I left it in here as what I'd like the code to look like, but we can't actually do it because I _think_ bitfield_struct is not respecting the feature gate in the derive macro.

If you actually do that code, then it complains that there are too many bits being used.

Reply:
In this specific case, it doesn't matter since the bit was reserved before, but it's a good hint that maybe we should fix it correctly? 

I'm not sure yet how I want to handle new igvm revisions, if those are exposed as features but this would stop us from doing that.